### DISCUSSION TITLE
GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by vridhi-pages** [community/community#59796] [GitHub Education]
### Hi everyone,

I have applied for student developer pack and got approved on 28 June 2023, but I am unable to access the features neither received any mail.
What to do?
How to contact GitHub?
Please help.

**Comment by Omrigotlieb** [comment]
Check your spam folder: Your approval email might be there.
Check your application status: You can do this on GitHub Education's site.
Contact GitHub: If still no luck, reach out to them. You can find their contact form on their site: https://support.github.com/contact , or tweet them at @github.

**Reply by vridhi-pages** [reply]
Thank you very much.
I will try to reach them

**Reply by Omrigotlieb** [reply]
Sure thing! If these steps work for you, please mark this query as answered. If you need more help, feel free to ask.
